scripts/docker/docker_bringup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p /home/${USER}/workspace
+HOST_UID=${UID} HOST_GID=${GID} docker compose up -d
+
+docker exec -it tt-metal bash
```

Comment:
does the `-v` parameter work here? If so, we should give user ability to mount their own folder(s)

Reply:
No, the -v will not work here but users do have the ability to mount their own folders in the docker-compose.yaml file underneath the 'volumes' section

The flow should be as such
user edits docker-compose.yaml - volumes section

user runs ./docker_bringup.sh

Reply:
Will add small comment in the docker-compose file to indicate to users to mount files here.

Reply:
update removed docker-compose and all references to it